src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -35,6 +37,14 @@
     public HelpWindow(Stage root) {
         super(FXML, root);
         helpMessage.setText(HELP_MESSAGE);
+        root.addEventFilter(KeyEvent.KEY_PRESSED, event -> {
```

Comment:
Actually I'm not sure if its a good idea to make the application close when the escape key is pressed. There is already a command "exit" that would do it. Especially when the popup window also closes when the escape key is pressed, users may end up accidentally exit the application if they accidentally if they hit escape key twice.

Reply:
My understanding is that the `HelpWindow` uses a different `Stage` from the `MainWindow` (see [line 54](https://github.com/AY1920S1-CS2103T-T10-4/main/blob/4ebd6dd50af5f1477fa68a18728275da1b8e29f7/src/main/java/seedu/address/ui/HelpWindow.java#L54) of `HelpWindow.java`). I believe this means that the escape keybind can only trigger when the popup is focused and subsequently only hides the the popup itself.

I have tried quickly hitting the escape key twice with these changes applied and only the `HelpWindow` popup closed.